core/src/main/scala/com/raphtory/deployment/Raphtory.scala

```diff
@@ -129,6 +112,7 @@ object Raphtory {
   def createGraphBuilder[T: ClassTag](
       builder: GraphBuilder[T]
   ): Unit = {
+    val scheduler        = new MonixScheduler()
```

Comment:
Should this be a singleton object? Which means unique per JVM. Currently, new will create a new scheduler per component.

Reply:
I am experimenting with this at the moment. If we have a separate scheduler per deployment we can kill all the threads at a top-level.